addons/website_sale/tests/test_website_editor.py

```diff
@@ -184,3 +185,29 @@ def test_resequence_images(self):
                     images[1].id,
                     'first',
                 )
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteSaleEditor(HttpCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.env['res.users'].create({
+            'name': 'Restricted Editor',
+            'login': 'restricted',
+            'password': 'restricted',
+            'groups_id': [Command.set([
+                cls.env.ref('base.group_user').id,
+                cls.env.ref('sales_team.group_sale_manager').id,
+                cls.env.ref('website.group_website_restricted_editor').id
+            ])]
+        })
```

Comment:
For tracking purpose (in case someone is wondering), I'd like to make most of our tour (when possible) run in restricted editor instead of admin.
This will come with a common test class, encapsulating this and using existing user creation tools already used in some tests.
So, for now, we just do the same thing as in `test_02_restricted_editor`

Reply:
Note that it is not exactly the same as the sale manager right had to be added here :wink: To keep in mind when we'll share a restricted user across all tests :open_mouth: